src/model/SettingsModel.hxx

```diff
@@ -0,0 +1,31 @@
+#ifndef SETTINGS_MODEL_HXX_
+#define SETTINGS_MODEL_HXX_
+
+#include "DataModel.hxx"
+#include "Localization.hxx"
+
+struct LanguageChangeEvent
+{
+  const iLocalization &localization;
+};
+
+using SettingsModelNotification = TypeList<LanguageChangeEvent>;
+
+class SettingsModel : public DataModel<SettingsModelNotification>
+{
+public:
+  SettingsModel(UILoopMQ &mq);
+  const iLocalization &getLocalization() { return m_Localization; }
+  void setLanguage(std::string language);
+
+private:
+  std::string m_Language;
+  Localization m_Localization;
```

Comment:
Shouldn't this be a `iLocalizationPtr`? Why did you define iLocalization in the first place? Is it too support modded localizations?

Reply:
I expect SettingsModel will grow in future. One of its members is Localization. I created iLocalization as an interface,  that it exposes only needed interface outside the module. I think it simplifies usage and latter we can change Localization completely (no more MoFileReader) without it get noticed outside the ModelSettings class.

Reply:
ok then